cloud/defaults.go

```diff
@@ -31,6 +32,11 @@ const (
 	DefaultInternalLBIPAddress = "10.0.0.100"
 )
 
+const (
+	// DefaultInternalLBIPv6Address is the default internal load balancer ip address
+	DefaultInternalLBIPv6Address = "2001:1234:5678:9abc::100"
```

Comment:
Is this needed since the internal load balancer was disabled for ipv6?

Reply:
it's not but I'm about to use it to enable internal LB for private cluster (WIP in another branch) so I'd rather just keep it